### DISCUSSION TITLE
Autocorrelation from neural activities

### DISCUSSION THREAD

**Discussion by AlbertXTang** [AdaptiveMotorControlLab/CEBRA#144] [Q&A]
Hi, I would like to ask a question about the cebra-behavior embedding derived from multi-session training. We know that there is usually strong autocorrelation from neural activities we measured. Does this kind of autocorrelation still exist in the cebra-behavior embedding after successful multi-session training? If it does, does the autocorrelation share across different sessions or subjects? Thank you!

**Comment by stes** [comment]
Hi @AlbertXTang , answering this depends a bit on how you record the data, and how you plan to use CEBRA. Depending on the labels you use, it is possible to break autocorrelation in the data, but also possible to keep it. It strongly depends on how the labels used for training relate to the neural data (and most importantly, if this relationship is linear or non-linear).

Could you expand more, or does this address your question? Is it a desired feature for you to retain the auto-correlation, or are you interested in features that are uncorrelated over time?

**Reply by AlbertXTang** [reply]
Thank you! I will examine the labels used for training for clarification.
